test/dotnet-test.UnitTests/GivenATestCommand.cs

```diff
@@ -14,10 +14,13 @@ namespace Microsoft.Dotnet.Tools.Test.Tests
     public class GivenATestCommand
     {
         private static readonly string ProjectJsonPath = Path.Combine(
-                AppContext.BaseDirectory,
+                Directory.GetCurrentDirectory(),
```

Comment:
I hadn't corrected the content path in project.json which was causing this to trip. Fixed it.